app/test/block_size_test.go

```diff
@@ -154,8 +177,11 @@ func (s *IntegrationTestSuite) TestMaxBlockSize() {
 			for _, hash := range hashes {
 				// TODO: reenable fetching and verifying proofs
 				resp, err := queryTx(val.ClientCtx, hash, false)
-				require.NoError(err)
-				require.NotNil(resp)
+				assert.NoError(err)
+				assert.NotNil(resp)
+				if resp == nil {
+					continue
+				}
```

Comment:
[question] is this conditional ever execution? If `resp == nil` were true, I would expect the assertion on the line above to fail.

Reply:
assert fails, but doesn't stop the test, so when we try to access the resp below it will panic